packages/manager/modules/nutanix/src/dashboard/general-info/routing.js

```diff
@@ -0,0 +1,10 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('nutanix.dashboard.general-info', {
+    url: '/general-info',
```

Comment:
Is there any reason to no prefer using `dashboard` over `general-info`?

Reply:
Dashoard is a parent route. It has general info and nodes route. Answer to your question, its been used.

Reply:
Why not leave an empty url for general-info ? That way customer lands on a dashboard when accessing /dashboard